packages/web/src/core/utils/web3-signature.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Module dependencies.
+ */
+
+import { Signer, utils } from 'ethers';
+import toLower from 'lodash/toLower';
+
+/**
+ * `fractalSignMessage`.
+ */
+
+const fractalSignMessage =
+  process.env.NEXT_PUBLIC_FRACTAL_KYC_SIGN_MESSAGE ||
```

Comment:
IMO this shouldn't be in an environment variable. when we have the copy we update, make sense